message/validation/validation.go

```diff
@@ -259,31 +249,23 @@ func (mv *messageValidator) getCommitteeAndValidatorIndices(msgID spectypes.Mess
 		operators = append(operators, c.Signer)
 	}
 
-	return CommitteeInfo{
-		operatorIDs: operators,
-		indices:     []phase0.ValidatorIndex{validator.BeaconMetadata.Index},
-		committeeID: validator.CommitteeID(),
-	}, nil
+	indices := []phase0.ValidatorIndex{validator.BeaconMetadata.Index}
+	return newCommitteeInfo(validator.CommitteeID(), operators, indices), nil
 }
 
-func (mv *messageValidator) consensusState(messageID spectypes.MessageID) *consensusState {
+func (mv *messageValidator) consensusState(messageID spectypes.MessageID, committee []spectypes.OperatorID) *consensusState {
```

Comment:
Perhaps I lost a bit the track, but how are we differentiating the consensusState for a certain PeerID?
Is this being separated on a higher level than here?
I ask this because different peers can't share the same state

Reply:
@MatheusFranco99 we check peer ID just to check if it's our message, we don't use it for anything other than that